pkg/lib/bundle/validate.go

```diff
@@ -99,15 +99,43 @@ func (i imageValidator) ValidateBundleFormat(directory string) error {
 		validationErrors = append(validationErrors, err)
 	}
 
-	// Validate annotations.yaml
-	annotationsFile, err := ioutil.ReadFile(filepath.Join(annotationsDir, AnnotationsFile))
+	// Validate annotations file
+	files, err := ioutil.ReadDir(metadataDir)
 	if err != nil {
-		fmtErr := fmt.Errorf("Unable to read annotations.yaml file: %s", err.Error())
-		validationErrors = append(validationErrors, fmtErr)
-		return NewValidationError(validationErrors)
+		validationErrors = append(validationErrors, err)
+	}
+
+	// Look for the metadata and manifests sub-directories to find the annotations file
+	fileAnnotations := &AnnotationMetadata{}
+	dependenciesFile := &registry.DependenciesFile{}
+	for _, f := range files {
+		err = registry.DecodeFile(filepath.Join(metadataDir, f.Name()), fileAnnotations)
+		if err != nil || fileAnnotations.Annotations == nil {
+			continue
+		}
+
+		err = parseDependenciesFile(filepath.Join(metadataDir, f.Name()), dependenciesFile)
+		if err != nil || len(dependenciesFile.Dependencies) < 1 {
+			continue
+		} else {
+			i.logger.Info("Found dependencies file")
+		}
 	}
 
-	var fileAnnotations AnnotationMetadata
+	if len(dependenciesFile.Dependencies) < 1 {
+		i.logger.Info("Could not find dependencies file")
+	}
+
+	if fileAnnotations.Annotations == nil {
+		validationErrors = append(validationErrors, fmt.Errorf("Could not find annotations file"))
```

Comment:
Is there supposed to be a `return` here? The next line logs that the annotations file was found.

Reply:
It was supposed to be an `else` statement there. Fixed.